File: source/PointOfSalesV2.Entities/Warehouse.cs


namespace PointOfSalesV2.Entities
{
    public class Warehouse : CommonData
    {
        public long BranchOfficeId { get; set; }
        [MaxLength(50)]
        public string Code { get; set; }
        [MaxLength(100)]
        [Translate]
        public string Name { get; set; }

        public List<Inventory> Inventory { get; set; }
        [ForeignKey("BranchOfficeId")]
        public BranchOffice BranchOffice { get; set; }
    }
}

File: source/PointOfSalesV2.FrontEnd/ClientApp/src/app/@core/data/warehouse.ts
import { BaseModel } from './baseModel';
import { BranchOffice } from './branchOffice';


export class Warehouse extends BaseModel {

     branchOfficeId :number = 0;
     code:string = '';
     name :string = '';
     inventory :any[] = [];
     branchOffice :BranchOffice | null = null;

}
